lib/galaxy/tools/deps/resolvers/__init__.py

```diff
@@ -22,10 +25,11 @@ class DependencyResolver(Dictifiable, object):
     # resolution.
     disabled = False
     resolves_simple_dependencies = True
+    config_options = {}
     __metaclass__ = ABCMeta
```

Comment:
FYI this is not Python3-safe, the [@six.add_metaclass(ABCMeta)](https://pythonhosted.org/six/#six.add_metaclass) decorator should be used instead. There are a few more instances in this file and elsewhere in `lib/galaxy/`, I will take care of fixing them all.